test/unit/test_easy_logging.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+
+import os.path
+
+import pytest
+import tomlkit
+
+from snowflake.connector import EasyLoggingConfigPython
+from snowflake.connector.constants import CONFIG_FILE
+
+
+@pytest.fixture(scope="function")
+def setup1():
+    file_path = os.path.dirname(CONFIG_FILE)
+    if os.path.exists(CONFIG_FILE):
+        with open(CONFIG_FILE) as f:
+            connection_content = tomlkit.parse(f.read())
+            connection_content["log"] = {"log_level": "wrong"}
+        with open(CONFIG_FILE, "w") as f:
+            f.write(tomlkit.dumps(connection_content))
+    else:
+        os.makedirs(file_path, exist_ok=True)
+        with open(CONFIG_FILE, "w") as f:
+            connection_content = {
+                "log": {
+                    "level": "wrong",
+                }
+            }
+            f.write(tomlkit.dumps(connection_content))
+    yield
```

Comment:
I've talked about this in other PRs, but you have to wrap the `yield` in a `try`/`finally`.  The reason being that the `yield` hands execution over to the context manager calling this fixture, but if any exceptions occur in the both of the context manager, it will pop up out of this function and code after the `yield` will not get executed.  I'm not 100% sure how fixtures are invoked, but it's good practice anyway.  E.g.:

```python
def fixture():
    # Set everything up.
    try:
        yield
    finally:
        # Clean everything up.
```